static/js/search_suggestion.js

```diff
@@ -113,7 +113,7 @@ function get_stream_suggestions(last, operators) {
         const prefix = "stream";
         const highlighted_stream = hilite(regex, stream);
         const verb = last.negated ? "exclude " : "";
-        const description = verb + prefix + " " + highlighted_stream;
+        const description = verb + prefix + "&nbsp;" + highlighted_stream;
```

Comment:
What's the motivation for this change?  I mostly want to make sure we're not at risk of breaking other search examples.

Reply:
Oh, I see it's in the commit message; I still don't understand, though -- are you fixing a problem created by other changes in this PR, or something that was present previously?